README.md

```diff
@@ -94,12 +94,25 @@ airflow db init
 Create an SQLite database for the example to run with and run the DAG:
 ```
 sqlite3 /tmp/sqlite_default.db "VACUUM;"
-airflow dags test calculate_top_animations `date --iso-8601=seconds`
+airflow dags test calculate_popular_movies `date -Iseconds`
```

Comment:
It might be worth updating on line 43 to also have the same name

Reply:
L43 is the name of the dag file though, and this is the dag_id

Reply:
I know - but it can simplify to the end-user keeping them with the same value :)

Reply:
fixed :)